Algebraic_kernel_d/doc/Algebraic_kernel_d/Algebraic_kernel_d.txt

```diff
@@ -325,27 +325,27 @@ efficiency.
 
 The following example illustrates the construction of `AlgebraicKernel_d_1::Algebraic_real_1` 
 using `AlgebraicKernel_d_1::Construct_algebraic_real_1`:
-<SMALL>\cgalExample{Algebraic_kernel_d/Construct_algebraic_real_1.cpp}</SMALL>
+\cgalExample{Algebraic_kernel_d/Construct_algebraic_real_1.cpp}
```

Comment:
Could you please explain again why this is needed, and only for this file?

Reply:
I only found this construct only in this file, all other files straight away used `\cgalExample`.

Reply:
I actually miss-read the diff and though you were adding the SMALL. I indeed don't see why we have it. Thanks!